build-release

```diff
@@ -5,12 +5,20 @@
 # ARCHITECTURE: architecture for the build
 # ISSUE_NUMBER: the number of the issue that triggered the build
 # DRY_RUN_PARAM: parameter that to use dry-run
+# ONLY_BUILD_TOOLCONF: if True, tells me to only build cmssw-tool-conf 
 
 HERE=$WORKSPACE/build
 BUILD_DIR=$HERE/$CMSSW_X_Y_Z-build
 mkdir -p $BUILD_DIR
 QUEUE=$CMSSW_QUEUE
 
+if [ "X$ONLY_BUILD_TOOLCONF" = Xtrue ]; then
+  POST_ACTION='TOOL_CONF_BUILDING'
+else
+  POST_ACTION='BUILDING'
+fi
+
+
 git clone git@github.com:cms-sw/cmsdist.git $BUILD_DIR/CMSDIST
```

Comment:
ah OK ..... I thought we are going to reuse the build area created by 
the tool-conf (to avoid downloading the RPMs again). It is true that if 
we reuse the build area then a release build might have to wait as the 
machine might be busy building ib or validation.
So who is going to cleanup the tool-conf build area?

On 03/04/2015 03:30 PM, David Mendez wrote:
>
> In build-release 
> <https://github.com/cms-sw/cms-bot/pull/362#discussion_r25777744>:
>
> >
> >  HERE=$WORKSPACE/build
> >  BUILD_DIR=$HERE/$CMSSW_X_Y_Z-build
> >  mkdir -p $BUILD_DIR
> >  QUEUE=$CMSSW_QUEUE
> >
> > +if [ "X$ONLY_BUILD_TOOLCONF" = Xtrue ]; then
> > +  POST_ACTION='TOOL_CONF_BUILDING'
> > +else
> > +  POST_ACTION='BUILDING'
> > +fi
> > +
> > +
> >  git clone git@github.com:cms-sw/cmsdist.git $BUILD_DIR/CMSDIST
>
> It deletes the workspace before the build starts. And also it can 
> choose a different machine for the build. Jenkins can assign 
> cmsbuild13 to build tool-conf and then cmsbuil15 to build cmssw.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/cms-sw/cms-bot/pull/362/files#r25777744>.
>